proto/desmos/posts/v1beta1/query.proto

```diff
@@ -51,49 +51,18 @@ service Query {
 
 // ___________________________________________________________________________________________________________________
 
-// QueryPostsParams represent the parameters used to query for posts
-message QueryPostsParams {
-  uint64 page = 1;
-  uint64 limit = 2;
-
-  string sort_by = 3;
-  string sort_order = 4;
-
-  string parent_id = 5 [
-    (gogoproto.customname) = "ParentID",
-    (gogoproto.jsontag) = "parent_id",
-    (gogoproto.moretags) = "yaml:\"id\""
-  ];
-  google.protobuf.Timestamp creation_time = 6 [ (gogoproto.stdtime) = true ];
-  string subspace = 7;
-  string creator = 8;
-  repeated string hashtags = 9;
-}
-
-// ___________________________________________________________________________________________________________________
-
 // QueryPostsRequest is the request type for the Query/Posts RPC method.
 message QueryPostsRequest {
-  string sort_by = 1;
-  string sort_order = 2;
-
-  string parent_id = 3 [
-    (gogoproto.jsontag) = "parent_id",
-    (gogoproto.moretags) = "yaml:\"id\""
-  ];
-  ;
-  google.protobuf.Timestamp creation_time = 4 [ (gogoproto.stdtime) = true ];
-  string subspace = 5;
-  string creator = 6;
-  repeated string hashtags = 7;
+  
+  string subspace = 1;
```

Comment:
```suggestion
  // Subspace must contain the ID of the subspace to which query the posts for. 
  // Providing an empty value will return an error.
  string subspace = 1;
```

Reply:
@RiccardoM Using `subspace_id` instead would be better and explicit.

Reply:
> @RiccardoM I think using `subspace_id` instead may be better and more explicit. What do you think?

Yeah we can do that